### ISSUE TITLE
Scripts' and logs' hyper-parameters are different.

### ISSUE THREAD

**Issue by sseung0703** [he-y/filter-pruning-geometric-median#45]
In case of Resnet20-cifar10, scripts say learning rate is **0.01**. However logs say **0.1.**
Also, In paper you said that 40% means **"30% pruned by distance and 10% pruned by norm".**
However, the logs name is **"cifar10_resnet20_ratenorm0.7_ratedist0.1_varience2"**, 
and it says that **"'rate_dist': 0.1, 'rate_norm': 1.0"**.
which one is correct?

**Comment by he-y** [comment]
The log for resnet20 of your mentioned setting is not correct.
Please use the correct results of resnet20 in [this issue](https://github.com/he-y/filter-pruning-geometric-median/issues/26).
Sorry for the confusing results.

**Comment by sseung0703** [comment]
I want not just results but reproducing. But I'm very confused to set hyper-parameters. 
which one is correct **hyper-parameters??**

**Comment by he-y** [comment]
This is an example of correct hyper-parameters:
![image](https://user-images.githubusercontent.com/10541284/70514193-22a7db80-1b6e-11ea-9a22-e02d7b08891a.png)

**Comment by sseung0703** [comment]
Thank you for so fast answer.
